beater/config/sampling.go

```diff
@@ -90,7 +108,6 @@ func defaultTailSamplingConfig() TailSamplingConfig {
 	return TailSamplingConfig{
 		Enabled:               false,
 		ESConfig:              elasticsearch.DefaultConfig(),
-		DefaultSampleRate:     0.5,
```

Comment:
Good question! I left out a default because I'm not sure that there's a good one-size-fits-most default. Our agents don't sample anything by default, so it seems reasonable to me to require users to define policies that make sense for their application.

Reply:
fair point